lib/error.js

```diff
@@ -55,8 +64,8 @@ export const makeError = ({
 	const isError = Object.prototype.toString.call(error) === '[object Error]';
 	const shortMessage = isError ? `${execaMessage}\n${error.message}` : execaMessage;
 	const message = [shortMessage, stdio[2], stdio[1], ...stdio.slice(3)]
+		.map(messagePart => stripFinalNewline(serializeMessagePart(messagePart)))
```

Comment:
We are joining each part with newlines. If a part ends with a newline, it just results in a single newline, not two.